modules/integration/src/test/scala/scala/cli/integration/NativePackagerTests.scala

```diff
@@ -107,6 +105,15 @@ class NativePackagerTests extends ScalaCliSuite {
           )
         }
       }
+
+    // FIXME: building dmg package sometimes fails with:
+    // 'hdiutil: couldn't eject "disk2" - Resource busy'
+    if (TestUtil.isM1 || !TestUtil.isCI)
+      test("building dmg package") {
+        testBuildingDmgPackage()
+      }
+    else test("building dmg package".flaky) {
+      testBuildingDmgPackage()
```

Comment:
I think it might be confusing to set flakiness according to platform, as the other tests in this file would also be affected.
So if someone tagged something as `.flaky` in another PR, he might be taken by surprise.
We can try to refactor this later, I'll merge this for now.